contracts/AMM/Referrals.sol

```diff
@@ -59,6 +80,43 @@ contract Referrals is Initializable, ProxyOwned, ProxyPausable, ProxyReentrancyG
         }
     }
 
+    /// @notice set Referral fees
+    /// @param _referrerFeeDefault how much of a fee to pay to referrers
+    /// @param _referrerFeeSilver how much of a fee to pay to silver referrers
+    /// @param _referrerFeeGold how much of a fee to pay to gold referrers
+    function setReferrerFees(
+        uint _referrerFeeDefault,
+        uint _referrerFeeSilver,
+        uint _referrerFeeGold
+    ) external onlyOwner {
+        uint TWO_PERCENT = 0.02 * 1e18;
+        require(
+            _referrerFeeDefault < TWO_PERCENT && _referrerFeeSilver < TWO_PERCENT && _referrerFeeGold < TWO_PERCENT,
+            "Maximum referrer fee is 2%"
+        );
+        referrerFeeDefault = _referrerFeeDefault;
+        referrerFeeSilver = _referrerFeeSilver;
+        referrerFeeGold = _referrerFeeGold;
+    }
+
+    /// @notice adding/removing silver address depending on a flag
+    /// @param _silverAddress address that needed to be added as silver or removed from silver addresses
+    /// @param _flag adding or removing from silver addresses (true: add, false: remove)
+    function setSilverAddress(address _silverAddress, bool _flag) external onlyOwner {
```

Comment:
I would add tier as a parameter and have one method to set referreralTier

Reply:
As agreed not for now